definitions/tools/filter_vcf_max_sv_pop_freq.cwl

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "POPFREQ_AF filter"
+baseCommand: ["/usr/bin/perl", "/usr/bin/vcf_check.pl"]
+requirements:
+    - class: InlineJavascriptRequirement
+    - class: DockerRequirement
+      dockerPull: "mgibio/vep_helper-cwl:1.0.1"
+    - class: ResourceRequirement
+      ramMin: 4000
+arguments:
+    [{ valueFrom: $(inputs.vcf.path) },
+    { valueFrom: $(runtime.outdir)/annotated.af_filtered.vcf },
```

Comment:
This parameter to `vcf_check.pl` needs to match the output file from the command.
```suggestion
    { valueFrom: $(runtime.outdir)/annotated.max_sv_pf_filtered.vcf },
```